src/Integration/YoastWooCommerceSeo.php

```diff
@@ -96,7 +96,7 @@ protected function add_value_option( array $value_options ): array {
 	 * @return mixed
 	 */
 	protected function get_mpn( $value, WC_Product $product ) {
-		if ( self::VALUE_KEY === $value ) {
+		if ( str_contains( $value, self::VALUE_KEY ) ) {
```

Comment:
Maybe I'm missing a commit, but I don't see this point addressed in the recent changes. It will still call a non existing function on PHP 7 with WordPress 5.8.

Reply:
You're right. I changing in other chunk on the code and missed this one.